packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/elements.ts

```diff
@@ -39,7 +39,7 @@ export const PropertyName = styled.span`
   `}
 `;
 
-export const PropertyValue = styled.span`
+export const PropertyValue = styled.span<{ css?: any }>`
```

Comment:
As mentioned before, let's instead wrap this in `styled`  rather than pass inlined styles through the css prop

Reply:
Done :)